src/controls/Jalousie.ts

```diff
@@ -41,6 +64,30 @@ export class Jalousie extends ControlBase {
                 // TODO: re-add: smartIgnore: false
             },
         );
+        this.addStateEventHandler(
+            control.states.position,
+            async (value: any) => {
+                if (typeof this.positionTarget === 'number') {
+                    // Below, the actual command ('up' or 'down') is irrelevant but we
+                    // need to know the direction for target test.
+                    if (this.positionTarget > 0) {
+                        // Going down
+                        if (value >= this.positionTarget) {
+                            this.positionTarget = undefined;
+                            await this.sendCommand(control.uuidAction, 'down');
```

Comment:
Is there a reason for using "down" instead of "DownOff" as in the original code?
Same below for "up" vs. "UpOff".

Reply:
Yes. `DownOff` and `UpOff` actually cause direction to reverse a little. Sending `up` or `down` just stops as it should.